pkg/cli/cmd/topaz/startrun.go

```diff
@@ -163,8 +163,10 @@ func getVolumes(cfg *config.Loader) ([]string, error) {
 	})
 
 	volumes := []string{
-		fmt.Sprintf("%s:/config:ro", cc.GetTopazCfgDir()),        // manually attach the configuration folder
-		fmt.Sprintf("%s:/root/.policy:ro", dockerx.PolicyRoot()), // manually attache policy store
+		fmt.Sprintf("%s:/config:ro", cc.GetTopazCfgDir()), // manually attach the configuration folder
+	}
+	if dockerx.PolicyRoot() != "" {
```

Comment:
Should mounting the policy root be restricted to the case when loading a local policy?

Reply:
Updated volume mount conditions to check the existence of the policies root folder and the configuration of the local policy image